yarn-project/sequencer-client/src/block_builder/solo_block_builder.test.ts

```diff
@@ -356,7 +356,7 @@ describe('sequencer/solo_block_builder', () => {
         expect(contractTreeAfter.root).toEqual(expectedContractTreeAfter);
         expect(contractTreeAfter.size).toEqual(BigInt(totalCount));
       },
-      10000,
```

Comment:
Are the tests significantly slower now?

Reply:
It takes 2x time to run this test (most probably since number of nullifiers have increased 4x). Showed around 22s on mainframe to run this test so increased the timeout to 30s.